src/app/projects/[ownerType]/[org]/[project]/cycles/[cycle]/page.js

```diff
@@ -259,6 +162,93 @@ export async function prepareData(params) {
   }
 
   const projectData = responseData[userOrOrganization].projectV2.items.nodes
+  for (let i = 0; i < projectData.length; i++) {
+    let item = projectData[i];
+    if (item.fieldValues.nodes.find(fv => fv.field?.name === 'Kind')?.name === "Bet") {
+      let {id} = item.content
+      try {
+        let scopeItems = await graphql(
+          `query($nodeId: ID!){
+            node (id: $nodeId){
+                ... on Issue {
+                  id
+                  title
+                  trackedIssues(first:100) {
+                    nodes {
+                      id
+                      title
+                      url
+                      closed
+                      closedAt
+                      createdAt
+                      comments(last: 100) {
+                          nodes {
+                            body
+                            bodyText
+                            createdAt
+                            updatedAt
+                            url
+                            author {
+                              avatarUrl(size: 100)
+                              ... on User {
+                                name
+                                url
+                              }
+                            }
+                          }
+                        }
+                      timelineItems(
+                          last: 100,
+                          itemTypes: CLOSED_EVENT
+                        ) {
+                          nodes {
+                            ... on ClosedEvent {
+                              __typename
+                              url
+                              stateReason
+                              actor {
+                                avatarUrl(size: 100)
+                                ... on Actor {
+                                  login
+                                  url
+                                }
+                                ... on Bot {
+                                  login
+                                  url
+                                }
+                                ... on User {
+                                  name
+                                  url
+                                }
+                              }
+                            }
+                          }
+                        }
+                    }
+                  }
+                }
+              }
+            }`,
+            {
+              nodeId: id,
+              headers: {
+                authorization: `token ${process.env.GITHUB_TOKEN}`
+              }
+            }
+        )
+        if (scopeItems.node.trackedIssues?.nodes) {
```

Comment:
I see you're only using `trackedIssues`. That won't work with bets like this: https://github.com/asyncapi/studio/issues/661. None of the scopes are issues and maybe they don't deserve the effort of creating an issue. I wonder if we can also parse this kind of list.

Reply:
@fmvilas indeed. good idea to add them as well.
I'll also add pull requests.
I'm wondering if we push this change and we add support for it in a second iteration

Reply:
It should work now with simple tasks
<img width="1336" alt="Screenshot 2023-06-20 at 18 37 33" src="https://github.com/shapeitapp/shapeitapp/assets/554438/36a6250d-34d9-4ff9-a666-02202f70a492">